src/Compilers/Core/Portable/PEWriter/MetadataWriter.PortablePdb.cs

```diff
@@ -525,6 +535,24 @@ private static ImmutableArray<byte> SerializeBitVector(ImmutableArray<TypedConst
             return builder.ToImmutableAndFree();
         }
 
+        private static void SerializeTupleElementNames(BlobBuilder builder, ImmutableArray<TypedConstant> names)
+        {
+            builder.WriteInt32(names.Length);
+            foreach (var name in names)
+            {
+                WriteUTF16String(builder, (string)name.Value ?? string.Empty);
```

Comment:
NUL-terminated UTF8. Then the number of entries doesn't need to be serialized -- ```builder.WriteInt32(names.Length)``` isn't needed.